helpdesk_mgmt_category_team/controller/main.py

```diff
@@ -44,7 +44,7 @@ def submit_ticket(self, **kw):
                     request.env['ir.attachment'].sudo().create({
                         'name': c_file.filename,
                         'datas': base64.b64encode(data),
-                        'datas_fname': c_file.filename,
+                        # 'datas_fname': c_file.filename,
```

Comment:
In general, code changes should be done without commenting previous parts. Any specific reason for this? if not, simply remove this line.

Reply:
This field is removed in odoo 15.0

Reply:
Fine. But then remove it here as well please.

Reply:
It is already removed. This code here showing is already outdated.